pkg/pillar/types/zedmanagertypes.go

```diff
@@ -132,6 +133,36 @@ type AppInstanceStatus struct {
 	ErrorAndTimeWithSource
 }
 
+func (status AppInstanceStatus) LogCreate() {
+	logObject := base.NewLogObject(base.AppInstanceLogType, status.DisplayName,
+		status.UUIDandVersion.UUID, status.Key())
+	if logObject == nil {
+		return
+	}
+	logObject.CloneAndAddField("state", status.State).AddField("restart-in-progress", status.RestartInprogress).
+		AddField("purge-in-progress", status.PurgeInprogress).Infof("App instance status")
```

Comment:
Should the string at the end include "create"?

Reply:
will do